development/RolesOntology.owl

```diff
@@ -238,116 +238,7 @@
         <rdfs:comment xml:lang="en">Property that indicates who is a contributor to a project or cohort</rdfs:comment>
         <rdfs:label xml:lang="en">contributor</rdfs:label>
     </owl:ObjectProperty>
-    
-
-
-    <!-- https://w3id.org/enigma#hasDataAnalyst -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasDataAnalyst"/>
-    
-
-
-    <!-- https://w3id.org/enigma#hasDataCollector -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasDataCollector"/>
-    
-
-
-    <!-- https://w3id.org/enigma#hasDesigner -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasDesigner">
-        <rdfs:subPropertyOf rdf:resource="https://w3id.org/enigma#contributor"/>
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#Project"/>
-        <rdfs:range rdf:resource="https://w3id.org/enigma#Person"/>
-        <vann:example xml:lang="en">BipolarProject hasDesigner JohnSmith</vann:example>
-        <rdfs:comment xml:lang="en">Property that points to the person responsible for helping designing a project.</rdfs:comment>
-        <rdfs:label xml:lang="en">has designer</rdfs:label>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- https://w3id.org/enigma#hasGeneticDataAnalyst -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasGeneticDataAnalyst">
-        <rdfs:subPropertyOf rdf:resource="https://w3id.org/enigma#hasDataAnalyst"/>
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#Project"/>
-        <rdfs:range rdf:resource="https://w3id.org/enigma#Person"/>
-        <vann:example xml:lang="en">BipolarProject hasGeneticDataAnalyst JohnSmith</vann:example>
-        <rdfs:comment xml:lang="en">Describes what project someone is a genetic data analyst of</rdfs:comment>
-        <rdfs:label xml:lang="en">has genetic data analyst</rdfs:label>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- https://w3id.org/enigma#hasGeneticDataCollector -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasGeneticDataCollector">
-        <rdfs:subPropertyOf rdf:resource="https://w3id.org/enigma#hasDataCollector"/>
-        <rdfs:domain rdf:resource="https://w3id.org/enigma#Project"/>
-        <rdfs:range rdf:resource="https://w3id.org/enigma#Person"/>
-        <vann:example xml:lang="en">BipolarProject hasGeneticDataCollector JohnSmith</vann:example>
-        <rdfs:comment xml:lang="en">Describes what project someone collected genetic data for</rdfs:comment>
-        <rdfs:label xml:lang="en">has genetic data collector</rdfs:label>
-    </owl:ObjectProperty>
-    
-
-
-    <!-- https://w3id.org/enigma#hasImagingDataAnalyst -->
-
-    <owl:ObjectProperty rdf:about="https://w3id.org/enigma#hasImagingDataAnalyst">
```

Comment:
We used some of these properties for the author generation... it would be nice to double check